code/data_structures/src/list/singly_linked_list/operations/unclassified/union_intersection_in_list.c

```diff
@@ -3,34 +3,37 @@
 // Part of Cosmos by OpenGenus Foundation
 #include<stdio.h>
 #include<stdlib.h>
-#include<conio.h>
 /* Link list node */
-struct Node
+struct 
+Node
 {
     int data;
     struct Node* next;
 };
- 
+
 /* A utility function to insert a node at the beginning of 
-   a linked list*/
-void push(struct Node** head_ref, int new_data);
- 
+ *   a linked list*/
+void 
+push(struct Node** head_ref, int new_data);
+
 /* A utility function to check if given data is present in a list */
-bool isPresent(struct Node *head, int data);
- 
+bool 
+isPresent(struct Node *head, int data);
```

Comment:
I'm not convinced that putting types on separate lines like this in function prototypes is normal. I have mostly seen prototypes as single lines, with function definitions having the type on the line above.

Reply:
Thanks! fixed!